suite-native/module-authorize-device/src/screens/connect/ConnectingDeviceScreen.tsx

```diff
@@ -6,7 +6,7 @@ import { useNativeStyles, prepareNativeStyle } from '@trezor/styles';
 import { useDelayedNavigation } from '@suite-native/device';
 import { Translation } from '@suite-native/intl';
 
-import { ConnectDeviceSreenView } from '../components/ConnectDeviceSreenView';
+import { ConnectDeviceSreenView } from '../../components/connect/ConnectDeviceSreenView';
```

Comment:
nit: (same unnecessary long path in multiple files)
```suggestion
import { ConnectDeviceSreenView } from './ConnectDeviceSreenView';
```

Reply:
Hmm, it's actually correct I think, since I'm going from `screens/connect` to `components/connect`

Reply:
my bad 👌